cdk/PgStacInfra.ts

```diff
@@ -22,6 +29,7 @@ import {
 import { DomainName } from "@aws-cdk/aws-apigatewayv2-alpha";
 import { readFileSync } from "fs";
 import { load } from "js-yaml";
+import { PgBouncer } from "./PgBouncer";
```

Comment:
Sorry if I haven't been following previous discussions, was this something you were thinking about adding to eoapi-cdk that we'd import from there? Or if you have a link to that discussion somewhere, can you drop it here? 🙏🏼

Reply:
Yeah I think we should add this to eoapi-cdk, but I wanted to deploy it here for a while first to see how it goes. I can't think of any specific discussions, just that whenever I talk to Bitner about connecting serverless applications to a pgstac database he says we need to be using connection pooling.